packages/adapter-components/src/definitions/system/utils.ts

```diff
@@ -71,7 +71,10 @@ export const queryWithDefault = <T, K extends string = string>(
   defsWithDefault: DefaultWithCustomizations<T, K>,
 ): DefQuery<T, K> => {
   const query: DefQuery<T, K>['query'] = key =>
-    mergeSingleDefWithDefault(defsWithDefault.default, defsWithDefault.customizations[key])
+    mergeSingleDefWithDefault(
+      defsWithDefault.default,
+      defsWithDefault.customizations ? defsWithDefault.customizations[key] : undefined,
```

Comment:
nit: I think you can just
```suggestion
      defsWithDefault.customizations?.[key],
```

Reply:
typescript didnt like it

Reply:
weird, it's working for me (notice you need `?.` and not `?`)

Reply:
my bad, fixed it